src/lib/com/yetanalytics/xapipe/filter/concept.clj

```diff
@@ -57,57 +44,58 @@
                          ::contextCategoryActivityType
                          ::contextOtherActivityType
                          ::attachmentUsageType]))
-  :ret ::validator)
+  :ret ::t/template)
 
 (defn t-val
-  "take a map of determining properties and returns a persephone validator with
+  "take a map of determining properties and returns a template with
   the determining properties merged onto a mock template"
   [det-props]
-  (p/template->validator (merge mock-template det-props)))
+  ;; TODO: This function is now private, update
+  (merge mock-template det-props))
```

Comment:
It is now un-private (though it has been renamed to `compile-templates->validators` and now takes a coll).

Reply:
forgot to remove the todo, the call isn't even there anymore

Reply:
https://github.com/yetanalytics/xapipe/pull/68/commits/4adc694d35226074c8bc8795765ff56b651513ae